src/styles/_normalize.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
```

Comment:
I've been told to avoid the use of * as much as possible  I would apply it directly to your calendar__day